blogposts/2022/release-3.38.md

```diff
@@ -0,0 +1,77 @@
+publishDate: 2022-03-21T10:00-07:00
+description: 'Sourcegraph 3.38 introduces improved Notebooks, faster Code Intelligence for large repos, dependencies search, and custom file syntax highlighting.'
+tags: [blog, release]
+slug: 'release/3.38'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+changelogItems:
+  - description: Code Insights will now periodically clean up data series that are not in use. There is a 1 hour grace period where the series can be reattached to a view, after which all of the time series data and metadata will be deleted.
+    url: https://github.com/sourcegraph/sourcegraph/pull/32094
+    category: Code Insights
+  - description: 'Code Insights critical telemetry total count now only includes insights that are not frozen (limited by trial mode restrictions).'
+    url: https://github.com/sourcegraph/sourcegraph/pull/32529
+    category: Code Insights
+  - description: 'Notebooks from private enterprise instances can now be embedded in external sites by enabling the `enable-embed-route` feature flag.'
+    url: https://docs.sourcegraph.com/notebooks/notebook-embedding
+    category: Notebooks
+  - description: LSIF upload pages now include a section listing the reasons and retention policies resulting in an upload being retained and not expired.
+    url: https://github.com/sourcegraph/sourcegraph/pull/30864
+    category: Code Intelligence
+  - description: 'Code monitoring now has a Logs tab enabled as a [Beta feature](https://docs.sourcegraph.com/admin/beta_and_experimental_features). This lets you see recent runs of your code monitors and determine if any notifications were sent or if there were any errors during the run.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/32292
+    category: Admin
+  - description: 'Searching for the pattern `//` with regular expression search is now interpreted literally and will search for `//`. Previously, the `//` pattern was interpreted as our regular expression syntax `/<regexp>/` which would in turn be intrpreted as the empty string. Since searching for an empty string offers little practical utility, we now instead interpret `//` to search for its literal meaning in regular expression search.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/31520
+    category: Search
+---
+
+Sourcegraph 3.38 is now available! For this release, we introduced:
+
+## Code Insights is generally available
+
+[Code Insights](https://docs.sourcegraph.com/code_insights) became generally available on March 10th. Code Insights reveals high-level information about your codebase, based on both how your code changes over time and its current state. You can learn more about why we built Code Insights in [the announcement from our CEO](https://about.sourcegraph.com/blog/announcing-code-insights/).
```

Comment:
```suggestion
[Code Insights](https://docs.sourcegraph.com/code_insights) transforms any search query into customizable, visual dashboards in seconds. Code Insights surfaces high-level information about your codebase, based on both how your code changes over time and its current state. You can learn more about why we built Code Insights in [the announcement from our CEO](https://about.sourcegraph.com/blog/announcing-code-insights/).
```

@Joelkw - I removed the date and added an additional sentence to introduce Code Insights. I also changed "reveals" to "surfaces." "Reveals" has connotations of sharing secretive information and we don't want to imply Code Insights makes secret information available.

Reply:
Thanks @abschu ! Makes sense, I was just copying this from the copy that folks put together for the last post (if you think this is extra important, we may want to change it there too): https://about.sourcegraph.com/blog/release/3.37/